utils/create-breadcrumbs.ts

```
@@ -0,0 +1,89 @@
+    .replace(/\bOP Mainnet\b/gi, 'OP Stack')
+    .replace(/\bOptimism Mainnet\b/gi, 'OP Stack')
+    .replace(/\bOptimism mainnet\b/gi, 'OP Stack');
+}
+
+const updateBreadcrumbFile = async (filePath: string): Promise<void> => {
+  try {
+    const content = await fs.readFile(filePath, 'utf-8');
+    const { data: frontMatter, content: fileContent } = matter(content);
+
+    if (frontMatter.description &&
+        frontMatter.description.match(/\bOP Mainnet\b|\bOptimism Mainnet\b/gi) &&
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)